dnx/xunit.performance.run.dnx/project.json

```diff
@@ -0,0 +1,29 @@
+{
+  "version": "1.0.0-*",
+  "description": "xunit.performance.run.dnx Console Application",
+  "authors": [ "ericeil" ],
+  "tags": [ "" ],
+  "projectUrl": "",
+  "licenseUrl": "",
+
+  "dependencies": {
+    "Microsoft.DotNet.xunit.performance.run.core": "1.0.0-alpha-build0015",
```

Comment:
Consider whether this version number should be the "99.99.99-dev" version that we use elsewhere.
My guess is yes. For the nighly build, we used to have a task in the custom .msbuild file that replaced all occurrences of that version in all project.json files. It's disabled or commented out at the moment but we could add it back.